patterns/alz/templates/policies.bicep

```diff
@@ -1,6 +1,8 @@
 targetScope = 'managementGroup'
 
-@metadata({ message: 'The JSON version of this file is programatically generated from Bicep. PLEASE DO NOT UPDATE MANUALLY!!' })
+@metadata({
```

Comment:
Please reverse the changes. We will break this template up into smaller parts due to hitting ARM limits. Since the new policy isn't part of an initiative yet, it doesn't need to be deployed yet.